packages/devtools_app/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
nit: 2019?

Reply:
This package is not materially change from what was in package:devtools so keeping the year the same is right.